northstar/Cargo.toml

```diff
@@ -9,34 +9,44 @@ license-file = "LICENSE.md"
 [dependencies]
 async-stream = { version = "0.3.0", optional = true }
 async-trait = { version = "0.1.50", optional = true }
+base64 = { version = "0.13.0", optional = true }
 bincode = { version = "1.3", optional = true }
 bitflags = { version = "1.2", optional = true }
+byteorder = { version = "1.4", optional = true }
 bytes = { version = "1.0", optional = true }
 bytesize = { version = "1.0", optional = true }
 caps = { version = "0.5.1", optional = true }
 derive-new = { version = "0.5.9", optional = true }
 ed25519-dalek = { version = "1.0", optional = true }
 floating-duration = { version = "0.1.2", optional = true }
+fs_extra = { version = "1.2", optional = true }
 futures = "0.3.14"
 hex = { version = "0.4.3", optional = true }
+humanize-rs = "0.1.5"
```

Comment:
This should be also optional. Only npk feature depends on it. It's used for parsing the tmpfs size in the manifest.